lib/lev/routine.rb

```diff
@@ -193,6 +192,14 @@ def initialize
 
     def self.included(base)
       base.extend(ClassMethods)
+      enable_background_jobs(base)
+    end
+
+    def perform(*args, &block)
+      if defined?(Resque::Plugins::Status)
+        outputs = call(*args, &block).outputs
+        completed(outputs.job_data)
```

Comment:
`perform` used  by plain old resque without resque-status I believe?  so need the `call` to be outside of the `defined?` block...

```ruby
def perform(*args, &block)
  outputs = call(*args, &block).outputs
  completed(outputs.job_data) if defined?(Resque::Plugins::Status)
end
```